src/org/labkey/test/BaseWebDriverTest.java

```diff
@@ -2438,6 +2438,13 @@ public int getExportScriptIconCount(String chartParentCls)
         return exportIconLoc.findElements(getDriver()).size();
     }
 
+    // Note: Keep in sync with OntologyManager.getStandardConversionErrorMessage()
+    public String getConversionErrorMessage(Object value, String fieldName, Class<?> targetClass)
+    {
+        String fromType = (value instanceof String) ? "" : "(" + (value.getClass().getSimpleName() + ") ");
+        return "Could not convert value " + fromType + "'" + value + "' for field '" + fieldName + "'; expected type " + targetClass.getSimpleName();
+    }
+
```

Comment:
Please move this to `LabKeySiteWrapper`. BWDT should just be methods for test setup, teardown, failure handling, and the like (yes, there is much cleanup to do in this regard).
I would suggest making it static and putting it in some utility class but I don't see any great candidates for where to move it.

Reply:
Moved